mlir/lib/Conversion/SCFToControlFlow/SCFToControlFlow.cpp

```diff
@@ -15,7 +15,6 @@
 
 #include "mlir/Dialect/Arith/IR/Arith.h"
 #include "mlir/Dialect/ControlFlow/IR/ControlFlowOps.h"
-#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
```

Comment:
this conversion pass no longer needs to depend on llvm dialect, and other conversion dialects where this info is currently being lost can now propagate these attributes without depending on it either